report.js

```diff
@@ -1,30 +1,27 @@
 function generateSummary(core, metrics) {
-  body = '### Lava scan results\n\n';
+  var body = '### Lava scan results\n\n';
 
   if (Object.keys(metrics.vulnerability_count).length === 0) {
-    return body + `No vulnerabilities found!! 🎉🎉🎉\n\n`;
+    return body + `No vulnerabilities found!! 🎉\n\n`;
   }
 
   body += `| Severity | Findings |\n`;
   body += `|---|---:|\n`;
 
-  if (metrics.vulnerability_count["success"] > 0) {
-    body += `| :white_check_mark: success | ${metrics.scan_count["success"]} |\n`;
-  }
   if (metrics.vulnerability_count["critical"] > 0) {
-    body += `| :fire: critical | ${metrics.vulnerability_count["critical"]} |\n`;
+    body += `| 🟣 critical | ${metrics.vulnerability_count["critical"]} |\n`;
   }
   if (metrics.vulnerability_count["high"] > 0) {
-    body += `| :red_circle: high | ${metrics.vulnerability_count["high"]} |\n`;
+    body += `| 🔴 high | ${metrics.vulnerability_count["high"]} |\n`;
   }
   if (metrics.vulnerability_count["medium"] > 0) {
-    body += `| :large_orange_diamond: medium | ${metrics.vulnerability_count["medium"]} |\n`;
+    body += `| 🟡 medium | ${metrics.vulnerability_count["medium"]} |\n`;
   }
   if (metrics.vulnerability_count["low"] > 0) {
-    body += `| :large_blue_diamond: low | ${metrics.vulnerability_count["low"]} |\n`;
+    body += `| 🔵 low | ${metrics.vulnerability_count["low"]} |\n`;
   }
   if (metrics.vulnerability_count["info"] > 0) {
-    body += `| :sunny: info | ${metrics.vulnerability_count["info"]} |\n`;
+    body += `| 🟢 info | ${metrics.vulnerability_count["info"]} |\n`;
```

Comment:
I would use the same colors we use in Vulcan:

![image](https://github.com/adevinta/lava-action/assets/1223476/fd424165-c387-48c1-b725-61d1259a8abd)

Reply:
Ok, I used the same used by lava human report, probably you should change it too.

Reply:
Unfortunately, we don't have that much variety of colors in the terminal if we want to maximize compatibility:

![image](https://github.com/adevinta/lava-action/assets/1223476/8de3a96b-2805-4791-8336-c959d6415742)

We could use bright colors, but does not guarantee enough contrast depending on the color scheme of the user.

So we went with:

![image](https://github.com/adevinta/lava-action/assets/1223476/22577f9c-0eda-462d-926e-3338eeb1ede6)

You could argue that INFO could be blue, but it is hard to tell how different it will look compared to cyan.